### ISSUE TITLE
Button is not showing

### ISSUE THREAD

**Issue by DirtyGaming** [weleho/fivem-status-bot#5]
First of all, it's a simple yet amazing and well optimized script that's why I'm working on adding new stuff to it and will definitely contribute to this project
So far, I couldn't have the button shown on the webhook message and I tried several fixes for hundreds of times on a whole night to make it work but nothing..
I think  there is something wrong with the components table in the encoded message?
```lua
local message = json.encode({
		content = "This is a message with components",
		embeds = {
			{

				["color"] = Config.EmbedColor,
				["title"] = '**'..Config.ServerName..'**',
				["description"] = line1 .. '\n' .. line2,
				["footer"] = {
					["text"] = footer,
					["icon_url"] = 'https://i.imgur.com/IyUdG5Z.png',
				},
			}
		},

		components = {
			{
				["type"] = 1,
				["components"] = {
					{
						["type"] = 2,
						["label"] = 'TEST',
						["style"] = 5,
						["url"] = 'https://twitch.tv/aymtv',
					}
				},
			}
		}
	})
```
I did not make any edits to the core script if you were wondering
This is what I get on Discord:
![image](https://user-images.githubusercontent.com/29959456/125170425-e9e74280-e1a6-11eb-9edc-93f181224631.png)
I see no buttons

**Comment by weleho** [comment]
Yeah, discord have made modifications how components work in webhooks. Fixing when i have time!

![image](https://user-images.githubusercontent.com/57502552/125261843-5169d300-e30a-11eb-8903-ff62f7530da3.png)

**Comment by DirtyGaming** [comment]
> Yeah, discord have made modifications how components work in webhooks. Fixing when i have time!
> 
> ![image](https://user-images.githubusercontent.com/57502552/125261843-5169d300-e30a-11eb-8903-ff62f7530da3.png)

That's exactly what I was thinking when I looked it up but I have no idea on what an application-owned webhook is?

**Comment by sarmad-gaming** [comment]
same here

**Comment by siddharth220** [comment]
Has anyone found the solution to this?

**Comment by gerardojro** [comment]
Same issue here!

**Comment by Rangxxr** [comment]
any update?

**Comment by Kraleemil** [comment]
Same problem, any update?

**Comment by epicroleplay23** [comment]
Same issue

**Comment by epicroleplay23** [comment]
@DirtyGaming i try to add more like ip server , online because i cant can you help me.
